open-company/open-company-web
# Add sections

card: https://trello.com/c/bfMlBt1p

to test:
- try to add a section with the link at the top of the ToC
- try to add a section btw other sections and check these facts:
  - the section is added in the right place if the categories match :heavy_check_mark:
  - the section is added at the top if the categories don't match :heavy_check_mark:
  - the category is added if it was empty :heavy_check_mark:
- check CI or run `lein test-all!`
- check the code
- merge

_NB_: saving a new added section is not part of this branch. Also when you select a new section and it gets added to the current list a js error is fired, i was unable to avoid this but the component that errors is unmounted right after so it should not be a problem. ping me for more info if you need
